examples/qml_features/rust/src/custom_base_class.rs

```diff
@@ -94,14 +80,26 @@ mod ffi {
                 return;
             }
 
-            self.as_mut().begin_remove_rows(index, index);
+            self.as_mut()
+                .begin_remove_rows(&QModelIndex::default(), index, index);
             self.as_mut().vector_mut().remove(index as usize);
             self.as_mut().end_remove_rows();
         }
     }
 
     // QAbstractListModel implementation
     impl qobject::CustomBaseClass {
+        cxx_qt::inherit! {
```

Comment:
Add an entry into the CHANGELOG about this :-)

Reply:
Done.
Also added documentation to the cxx-qt crate + book